doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -60,7 +60,37 @@ The issues in this section are related to :ref:`protocols`.
 Amazon Sidewalk
```

Comment:
Just adding this here, please feel free to move this text to the most relevant section

```suggestion
 .. rst-class:: v2-8-0

NCSDK-28152: TPIU Trace Signal Skew on the nRF54H20 SoC
    Segger TPIU tracing on the nRF54H20 DK encounters instability due to a minimal skew between clock and data signals.
    This skew causes unreliable trace performance, resulting in intermittent data capture.

  **Workaround:** Adjusting the TPIU trace delay on SEGGER's J-Trace Pro can improve signal stability, enhancing ETM capture reliability.
                  For more information on adjusting trace timing, refer to the SEGGER Wiki's "Adjusting Trace Timings and General Troubleshooting" section.
                  
Amazon Sidewalk 
```

Also, links.txt need a link to this page: https://wiki.segger.com/Adjusting_trace_timings_and_general_troubleshooting#Type_3:_Trace_clock_delayed

Reply:
@shanthanordic Please check this text

Reply:
Added to Other issues section.

Reply:
that is fine.